tests/perfTests.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+import sys
+import time
+import array
+
+import wx
+wx.App()
+
+from skimage.transform import resize
+from skimage.util import img_as_float
+
+import numpy as np
+
+from PIL import Image
+
+
+# CONSTANTS #
+SRC_W = 640
+SRC_H = 480
+SRC_D = 3
+OFFSET_X = 400
+OFFSET_Y = 240
+
+IMG_W = 200
+IMG_H = 66
+IMG_D = 3
+
+
+# ORIGINAL take_screenshot() #
+def take_screenshot():
+    screen = wx.ScreenDC()
+    size = screen.GetSize()
+    bmp = wx.Bitmap(size[0], size[1])
+    mem = wx.MemoryDC(bmp)
+    mem.Blit(0, 0, size[0], size[1], screen, 0, 0)
+    return bmp.GetSubBitmap(wx.Rect([0,0],[SRC_W,SRC_H]))
+
+
+# MODIFIED take_screenshot() #
+def take_screenshot_direct_screen_rect():
+    screen = wx.ScreenDC()
+    bmp = wx.Bitmap(SRC_W, SRC_H)
+    mem = wx.MemoryDC(bmp)
+    mem.Blit(0, 0, SRC_W, SRC_H, screen, OFFSET_X, OFFSET_Y)
+    return bmp
+
+
+# ORIGINAL prepare_image() #
+def prepare_image(img):
+    buf = img.ConvertToImage().GetData()
+    img = np.frombuffer(buf, dtype='uint8')
+
+    img = img.reshape(SRC_H, SRC_W, SRC_D)
+    img = resize(img, [IMG_H, IMG_W])
+
+    return img
+
+
+# MODIFIED prepare_image() #
+arr = array.array('B', [0] * (SRC_W * SRC_H * SRC_D));
+def prepare_image_with_PIL(img):
+    img.CopyToBuffer(arr)
+    img = np.frombuffer(arr, dtype=np.uint8)
+
+    img = img.reshape(SRC_H, SRC_W, SRC_D)
+
+    im = Image.fromarray(img)
+    im = im.resize((IMG_W, IMG_H))
+
+    im_arr = np.frombuffer(im.tobytes(), dtype=np.uint8)
+    im_arr = im_arr.reshape((IMG_H, IMG_W, IMG_D))
+
+    return img_as_float(im_arr)
+
+
+# CALL ORIGINAL #
+def call_original():
+    bmp = take_screenshot()
+    vec = prepare_image(bmp)
+
+
+# CALL MODIFIED #
+def call_modified():
+    bmp = take_screenshot_direct_screen_rect()
+    vec = prepare_image_with_PIL(bmp)
+
+
+# MAIN #
```

Comment:
I'd prefer to remove the comments that simply mimic the function name in this file